app/javascript/components/Sighting.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import RatingBar from './RatingBar'
+import iconData from '../constants/iconData'
+
+const Sighting = props => {
+
+  return(
+    <div className='sightingTile row'>
```

Comment:
I'd like to see all of your classNames be consistent, ideally with dashes. so for example, `sighting-tile` instead of `sightingTile`, `cryptid-pic` instead of `cryptid_pic`, etc!